lft/consensus/messages/data.py

```diff
@@ -60,7 +60,7 @@ def __hash__(self):
 
 class DataVerifier(ABC):
     @abstractmethod
-    async def verify(self, data: 'Data'):
+    async def verify(self, prev_data: 'Data', data: 'Data'):
         raise NotImplementedError
```

Comment:
How about giving a information that this method expects `Exception` when failed in verification?

```suggestion
        """Verify Data.
        
        :param prev_data:
        :param data:
        :raises:
            Exception:
        """
        raise NotImplementedError
```